index.html

```diff
@@ -65,54 +77,70 @@ <h4>Skills</h4>
+            </div>
             </div>
             <div class="works" id="work-2">
-                <img src="Works/work2.png" alt="Second work image">
-                <h4>Microverse Project 1</h4>
-                <p>This is a project with CSS. It was an introduction to the 
-                    use of linters and the process required for a code review.
-                </p>
-                <ul class="project-languages">
-                    <li class="project-languages-1">html</li>
-                    <li class="project-languages-1">css</li>
-                </ul>
-                <div class="n1"><button>See Project</button></div>
+                <div class="desktop-layout" id="second-works">
+                    <img src="Works/work2.png" alt="Second work image">
+                    <div class="desktop-layout-1">
+                        <h4>Microverse Project 1</h4>
+                        <p>This is a project with CSS. It was an introduction to the 
+                            use of linters and the process required for a code review.
+                        </p>
+                        <ul class="project-languages">
+                            <li class="project-languages-1">html</li>
+                            <li class="project-languages-1">css</li>
+                        </ul>
+                        <div class="n1"><button>See Project</button></div>
+                    </div>
+                </div>
             </div>
             <div class="works" id="work-3">
-                <img src="Works/work3.png" alt="Third work image">
-                <h4>Personal portfolio project</h4>
-                <p>This project was designed by Microverse to aid new
-                    students design their peronal portfolio. The basic languages
-                    for this project were HTML and CSS.
-                </p>
-                <ul class="project-languages">
-                    <li class="project-languages-1">html</li>
-                    <li class="project-languages-1">css</li>
-                </ul>
-                <div class="n1"><button>See Project</button></div>
+                <div class="desktop-layout">
+                    <img src="Works/work3.png" alt="Third work image">
+                    <div class="desktop-layout-1">
+                        <h4>Personal portfolio project</h4>
+                        <p>This project was designed by Microverse to aid new
+                            students design their peronal portfolio. The basic languages
+                            for this project were HTML and CSS.
+                        </p>
+                        <ul class="project-languages">
+                            <li class="project-languages-1">html</li>
+                            <li class="project-languages-1">css</li>
+                        </ul>
+                        <div class="n1"><button>See Project</button></div>
+                    </div>
+                </div>
```

Comment:
Figma templat
![Screenshot from 2022-04-30 21-13-45](https://user-images.githubusercontent.com/77842237/166117507-9a85c2ec-663b-4be6-b134-b7ff697e128b.png)
e

Your Design
![Screenshot from 2022-04-30 21-14-35](https://user-images.githubusercontent.com/77842237/166117525-8daf68d7-af60-4487-a4f5-51dff53f141c.png)

- [ ] Kindly the design of the projects are not in accordance with the design of Figma. the projects should start below the `see project` button. the background was almost but not quite right. make sure to open the cross lines in the background
- [ ] remember that You must replicate the design as much as possible (e.g., font, colors, images, tests, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).